### ISSUE TITLE
The alarm request doesn't work

### ISSUE THREAD

**Issue by msaravia98** [prey/prey-android-client#150]
When I sent a request to turn on the phone's alarm it gets queued but the request never reach to the destination device.

**Comment by patriciojofre** [comment]
Hi @msaravia98, can you trigger any other action? like get location?

**Comment by msaravia98** [comment]
Yes, I get the phone's location, then I send the alarm but it never gets to the phone (as I said it just got queued). The phone's location is correct, the problem is sending the alarm.
